config.php.example

```diff
@@ -5,10 +5,11 @@
  *
  */
 
+$environment = "dev"; // dev/test/prod
 $host       = "127.0.0.1";
 $username   = "root";
 $password   = "root";
-$dbname     = "dev";
+$dbname     = $environment."_cga";
```

Comment:
what does the _cga suffix represent?

Reply:
@chris7neves "CrsMgr Group-work Assistant"